include/regs.h

```diff
@@ -183,6 +183,8 @@
 #define R_ROOM_CULL_USED_ENTRIES                 iREG(88)
 #define R_ROOM_CULL_DEBUG_TARGET                 iREG(89)
 #define R_B_LABEL_DD                             WREG(0)
+#define R_PAUSE_OFFSET_VERTICAL                  WREG(2)
+#define R_PAUSE_OFFSET_DEPTH                     WREG(3) // Offset position of all pages away from the camera
 #if OOT_NTSC
```

Comment:
I get the symmetry argument but personally I like names that read "forwards", like "vertical offset" and "depth offset". same applies to a few member variables in PauseContext if we make this change